job.go

```diff
@@ -269,7 +269,6 @@ func (c *connection) connect(job *Job) error {
 	// be nice and don't use up too many connections for mere metrics
 	conn.SetMaxOpenConns(1)
 	conn.SetMaxIdleConns(1)
-	conn.SetConnMaxLifetime(job.Interval * 2)
```

Comment:
Should we remove this option in general or only if Azure or MySQL is used? I think it might be better to do it conditionally than removing it for everyone.

Reply:
Good point.

What about this?

```
conn.SetMaxOpenConns(1)
conn.SetMaxIdleConns(1)
// disable SetConnMaxLifetime if MySQL
if c.driver != "mysql" {
	conn.SetConnMaxLifetime(job.Interval * 2)
}
```

Reply:
Sounds good! I'd maybe add why it's disabled to the comment so future developers know why this special case is needed. Something like "Disable SetConnMaxLifetime if MySQL as it is causing issues with the mysql driver we are using. See #60".